### PR TITLE
Update benefits-limitations.md

### PR DESCRIPTION
**Pull request by JamelleWT**
@KishaT content pasted

### CODE REVIEW

File: pages/annuity-basics/benefits-limitations.md [hunk 1]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+## Annuity types
+We offer the following annuity options, or guaranteed monthly payments, through our annuity provider:
+Single life annuity and a joint life annuity with your spouse or with someone other than your spouse. 
+<div class="usa-alert usa-alert-info">
+<div class="usa-alert-body" markdown="1">
+The TSP annuity is not the same as the annuity you receive as part of your CSRS or FERS retirement package or military retired pay. Nor is it the same as the monthly payment option you may chose as one of the other full withdrawal options.
+{: .usa-alert-text}
+</div>
+</div>
+#### Single life annuity
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

File: pages/annuity-basics/benefits-limitations.md [hunk 2]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+A single life annuity provides a monthly amount only to you for as long as you live. It stops paying after your death.
+#### Joint life annuity
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

File: pages/annuity-basics/benefits-limitations.md [hunk 3]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+A joint life provides a monthly amount to you while you and the person with whom you choose to share your annuity (your "joint annuitant") are alive. When either you or your joint annuitant dies, the annuity will continue to pay monthly benefits to the survivor for the rest of his or her life.
+In most cases, participants choose their spouse as the joint annuitant. If you choose someone other than your spouse as your joint annuitant, he or she must be either a former spouse or someone with an insurable interest. This means that the person is financially dependent on you and could reasonably expect to benefit financially from your continued life. Blood relatives or adopted relatives (but not relatives by marriage) who are closer than first cousins are presumed to have an insurable interest in you.
+
+The amount of the monthly payment while you and your joint annuitant are alive and the amount of the payment to the survivor depend on whether you choose a 100% or a 50% annuity.
+ 
+##### 100% survivor annuity
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 4 header `####`

File: pages/annuity-basics/benefits-limitations.md [hunk 4]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+A joint life provides a monthly amount to you while you and the person with whom you choose to share your annuity (your "joint annuitant") are alive. When either you or your joint annuitant dies, the annuity will continue to pay monthly benefits to the survivor for the rest of his or her life.
+In most cases, participants choose their spouse as the joint annuitant. If you choose someone other than your spouse as your joint annuitant, he or she must be either a former spouse or someone with an insurable interest. This means that the person is financially dependent on you and could reasonably expect to benefit financially from your continued life. Blood relatives or adopted relatives (but not relatives by marriage) who are closer than first cousins are presumed to have an insurable interest in you.
+
+The amount of the monthly payment while you and your joint annuitant are alive and the amount of the payment to the survivor depend on whether you choose a 100% or a 50% annuity.
+ 
+##### 100% survivor annuity 
+The amount of the monthly annuity payment to the survivor is the same as the annuity payment made while you and your joint annuitant are alive.
+##### 50% survivor annuity
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 4 header `####`

File: pages/annuity-basics/benefits-limitations.md [hunk 5]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+The amount of the monthly annuity payment to the survivor is the same as the annuity payment made while you and your joint annuitant are alive.
+##### 50% survivor annuity 
+The amount of the monthly payment to the survivor - whether the survivor is you or your joint annuitant - is cut to 50% of the monthly annuity payment made while both you and your joint annuitant are alive. For example, if your monthly annuity payment is $600 while both you and your joint annuitant are alive, it will be reduced by half, to $300 per month, **when either you or your joint annuitant dies**.
+If you name a joint annuitant other than your spouse who is more than 10 years younger than you, you must choose a joint life annuity with the 50% survivor benefit.
+
+For more information about joint annuitants, refer to the Annuities section of the booklet [_Withdrawing Your TSP Account After Leaving Federal Service_](#). 
+## Annuity payment options
+Once you’ve chosen either a single life or a joint life annuity, you must decide whether you want to receive level or increasing payments. Review these carefully to select the options and features that best suit your needs.
+#### Level payments
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

File: pages/annuity-basics/benefits-limitations.md [hunk 6]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+With level payments, the amount of the monthly annuity payment remains the same from year to year.
+#### Increasing payments
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

File: pages/annuity-basics/benefits-limitations.md [hunk 7]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+With increasing payments, the amount of the monthly annuity payment can change each year on the anniversary date of the first payment. The amount of the change is based on the change in inflation, as measured by the consumer price index (CPI).
+ 
+Increases in your monthly payment cannot exceed 3% per year (even if the CPI is greater than 3%), but the monthly annuity payments cannot decrease.
+ 
+Be aware that if you select this option, your annuity payments will be smaller than if you had selected the level payments option; and your payments could potentially increase every year.
+You can choose increasing payments when you select a single life annuity or a joint life annuity with your spouse. You cannot choose increasing payments when the joint annuitant is not your spouse.
+
+## Providing for beneficiaries
+Two additional features are available to you if you want to provide payments to the beneficiary, or beneficiaries, you name in the annuity section of your TSP withdrawal form. Keep in mind that when you choose additional features, your monthly annuity payments will be less than they would have been had you not chosen any.
+#### Cash refund
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

File: pages/annuity-basics/benefits-limitations.md [hunk 8]
```diff
@@ -11,3 +11,71 @@ scripts:
...
+If you (and your joint annuitant, if applicable) die before the amount of your TSP balance used to purchase your annuity has been paid out, the remaining amount will be paid to your beneficiary, or beneficiaries, in a lump sum.
+You can add this feature if you purchase a single life or a joint life annuity, and with level or increasing payments.
+#### Ten-year certain
```
Comment (line47) [inline pages/annuity-basics/benefits-limitations.md]: This should be a level 3 header `###`

**Comment by KishaT** [comment]
@line47 I've made all of the header changes in this page.

**Pull request merged**